Dockerfile

```diff
@@ -63,4 +63,13 @@ RUN apt-get -y install debootstrap
 # Install shell check
 RUN apt-get -y --no-install-recommends install shellcheck
 
+# Install bindgen build tools
+RUN apt-get -y --no-install-recommends install llvm-dev libclang-dev clang
+
+# Install musl build tools
+RUN apt-get -y --no-install-recommends install musl-tools
+
+# Install headers required for musl build
+RUN curl https://musl.cc/x86_64-linux-musl-native.tgz | tar -xz  -C /usr/include/
```

Comment:
The docker file is platform independent. It needs to work on aarch64 as well. This will fail on aarch64.

Reply:
Fixed it properly this time among other things.